frontend/src/fixtures/coursesFixtures.js

```diff
@@ -0,0 +1,27 @@
+const coursesFixtures = {
+    oneCourses: [
+        { 
+            "quarter": "20222",
+            "courseId": "ANTH 2 ",
+            "title": "INTRO CULT ANTHRO",
+            "description": "The nature of culture: survey of the range of cultural phenomena.",
+            "objLevelCode": "U",
+            "subjectArea": "ANTH ",
+            "unitsFixed": 4,
+        }
+                    ],
+    oneCoursesLongDescription: [
+        { 
+            "quarter": "20222",
+            "courseId": "ANTH 2 ",
+            "title": "INTRO CULT ANTHRO",
+            "description": "The nature of culture: survey of the range of cultural phenomena, including material culture, social organization, religion, and other topics.he nature of culture: survey of the range of cultural phenomena, including material culture, social organization, religion, and other topicshe nature of culture: survey of the range of cultural phenomena, including material culture, social organization, religion, and other topics",
```

Comment:
A small copy-paste error I noticed. It would be great if you fixed this - you can either do it here or in a separate PR.
```suggestion
            "description": "The nature of culture: survey of the range of cultural phenomena, including material culture, social organization, religion, and other topics. The nature of culture: survey of the range of cultural phenomena, including material culture, social organization, religion, and other topics. The nature of culture: survey of the range of cultural phenomena, including material culture, social organization, religion, and other topics",
```